app/javascript/src/controllers/thumbnail_controller.js

```diff
@@ -1,12 +1,21 @@
 import { Controller } from "@hotwired/stimulus"
 
 export default class extends Controller {
-  static targets = [ ]
+  static targets = [ "mediaSliderThumb" ]
```

Comment:
I think I'd do this without a target and just get all the sibling tabs via `evt.target.closest('[role="tablist"]').querySelectorAll('[role="tab"]')`

Reply:
Done, this made for a bit of a simpler change, thanks. :-)